flink-runtime/src/test/java/org/apache/flink/runtime/dispatcher/DispatcherFailoverITCase.java

```diff
@@ -64,15 +66,19 @@
 /** An integration test for various fail-over scenarios of the {@link Dispatcher} component. */
 public class DispatcherFailoverITCase extends AbstractDispatcherTest {
 
-    private static final Time TIMEOUT = Time.seconds(1);
-
     private final BlockingQueue<RpcEndpoint> toTerminate = new LinkedBlockingQueue<>();
 
     @Before
     public void setUp() throws Exception {
         super.setUp();
         final CompletedCheckpointStore completedCheckpointStore =
                 new EmbeddedCompletedCheckpointStore();
+        if (ClusterOptions.isAdaptiveSchedulerEnabled(configuration)) {
+            // Align timeouts for adaptive scheduler with test-wide timeouts.
+            configuration.set(JobManagerOptions.RESOURCE_WAIT_TIMEOUT, Time.toDuration(TIMEOUT));
+            configuration.set(
+                    JobManagerOptions.RESOURCE_STABILIZATION_TIMEOUT, Time.toDuration(TIMEOUT));
+        }
```

Comment:
I think these changes are not needed since we provide enough slots to run the job with the desired parallelism.

Reply:
Yes, that was exactly the motivation, I wanted to set these to Long.MAX_VALUE, but got an overflow because we're converting it to nanoseconds in `AkkaInvocationHandler`.

`RESOURCE_WAIT_TIMEOUT` defaults to `5m`, so this should be save to remove 👍